plugins/home-react/README.md

```diff
@@ -0,0 +1,3 @@
+# Home React
```

Comment:
Are there any specific reasons not to extract [componentRenderers](https://github.com/backstage/backstage/tree/master/plugins/home/src/componentRenderers) as well? They may also be useful for other plugins

Reply:
They are not currently exported as far as i can tell, but can be added if there is a need

Reply:
Oh, right, I just noticed that they are exported as react extensions: here: https://github.com/backstage/backstage/blob/master/plugins/home/src/plugin.ts#L45-L73